active_directory_ldap/help.md

```diff
@@ -72,17 +87,33 @@ This action is used to add the specified Active Directory user.
 
 ##### Input
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|first_name|string|None|True|User's first name|None|
-|last_name|string|None|True|User's last name|None|
-|additional_parameters|object|None|False|Add additional user parameters in JSON format e.g. {'telephoneNumber': '(617)555-1234'}|None|
-|domain_name|string|None|True|The domain name this user will belong to, e.g. mydomain.com|None|
-|user_ou|string|Users|True|The OU that the user account will be created in|None|
-|logon_name|string|None|True|The logon name for the account|None|
-|account_disabled|string|true|True|Set this to true to disable the user account at creation|['true', 'false']|
-|password|password|None|True|The account's starting password|None|
-|user_principal_name|string|None|True|The users principal name, e.g. user@example.com|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|account_disabled|string|True|True|Set this to true to disable the user account at creation|['true', 'false']|True|
+|additional_parameters|object|None|False|Add additional user parameters in JSON format|None|{'telephoneNumber':'(617)555-1234'}|
+|domain_name|string|None|True|The domain name this user will belong to|None|example.com|
+|first_name|string|None|True|User's first name|None|John|
+|last_name|string|None|True|User's last name|None|Doe|
+|logon_name|string|None|True|The logon name for the account|None|jdoe|
+|password|password|None|True|The account's starting password|None|mypassword|
+|user_ou|string|Users|True|The OU that the user account will be created in|None|Users|
+|user_principal_name|string|None|True|The users principal name|None|user@example.com|
+
+Example input:
+
+```
+{
+  "account_disabled": "true",
+  "additional_parameters": "{'telephoneNumber':'(617)555-1234'}",
```

Comment:
Should this be an object? also single-quotes are not valid JSON if it was intended to ba JSON object.